hw/mcu/dialog/da1469x/include/mcu/da1469x_dma.h

```diff
@@ -43,15 +43,24 @@ extern "C" {
 #define MCU_DMA_PERIPH_SDADC            (13)
 #define MCU_DMA_PERIPH_NONE             (15)
 
+/* DMA burst mode can be one of following values */
+#define MCU_DMA_BURST_MODE_DISABLED     (0 << 13)
+#define MCU_DMA_BURST_MODE_4B           (1 << 13)
+#define MCU_DMA_BURST_MODE_8B           (2 << 13)
+
+/* DMA CTRL SRC/DST increment */
+#define MCU_DMA_CTRL_AINC        (1 << 5)
+#define MCU_DMA_CTRL_BINC        (1 << 4)
+
+/* DMA CTRL enable DREQ peripheral request */
+#define MCU_DMA_CTRL_DREQ_MODE   (1 << 3)
+
 /* DMA bus width can be one of following values */
-#define MCU_DMA_BUS_WIDTH_1B            (0)
```

Comment:
Those constant are used in the several packages so removing them is not best option

Reply:
reverted these changes.